docs/source/_static/index.js

```diff
@@ -0,0 +1,82 @@
+// Load jquery and jquery-ui
+var $ = require('jquery');
+require('jquery-ui');
+$.getQueryParameters = window.$.getQueryParameters;
+$.urldecode = window.$.urldecode;
+window.$ = window.jQuery = $;
+
+// Load styling
+require('jquery-ui/themes/smoothness/jquery-ui.min.css');
+require("jupyter-js-widgets/css/widgets.min.css");
+
+const widgets = require('jupyter-js-widgets');
+console.info('jupyter-js-widgets loaded successfully');
+
+class WidgetManager extends widgets.ManagerBase {
+  displayWidgetState(models, el) {
+    const displays = [];
+    for (var id of Object.keys(models)) {
+      if (this._models[id]) {
+        displays.push(this._models[id].then(model => {
+          return this.display_model({}, model, { el })
+        }));
+      } else {
+        displays.push(this.new_model({
+          model_id: id,
+          model_name: models[id].model_name,
+          model_module: models[id].model_module,
+        }, models[id].state).then(model => this.display_model({}, model, { el })));
```

Comment:
Hence whatever widget model in our script tag is displayed as a top-level thing in the hierarchy should  have an attribute "views" like [here](https://github.com/ipython/ipywidgets/blob/master/jupyter-js-widgets/examples/web2/widget_state.json).